### PR TITLE
stats for number of circuit breaker triggered

### PR DESCRIPTION
**Pull request by vr000m**
Fixes the last open issue proposed in #61.

### CODE REVIEW

File: webrtc-stats.html [hunk 1]
```diff
@@ -2264,6 +2265,15 @@
                 </p>
               </dd>
               <dt>
+                <dfn><code>circuitBreakerTriggered</code></dfn> of type <span class=
+                "idlMemberType"><a>unsigned long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Represents the number of times the circuit breaker is triggered for this particular 5-tuple. Ceasing transmission when a circuit breaker is triggered is defined in Section 4.5 of [[!RFC8083]]. The field MUST return undefined for user-agents that do not implement the circuit-breaker algorithm.
```
Comment (taylor-b) [inline webrtc-stats.html]: nit: Break up line

File: webrtc-stats.html [hunk 2]
```diff
@@ -2264,6 +2265,15 @@
                 </p>
               </dd>
               <dt>
+                <dfn><code>circuitBreakerTriggered</code></dfn> of type <span class=
```
Comment (taylor-b) [inline webrtc-stats.html]: bikeshed: `circuitBreakersTriggered`? Or `circuitBreakerTriggerCount`?

File: webrtc-stats.html (hunk 1 above)
Comment (vr000m) [inline webrtc-stats.html]: ok.

File: webrtc-stats.html (hunk 2 above)
Comment (vr000m) [inline webrtc-stats.html]: circuitBreakerTriggerCount is better, will change.

**Pull request merged**